tests/tests_input/simple_script.py

```diff
@@ -0,0 +1,5 @@
+import powerfactory
```

Comment:
why are .py files inputs or outputs of the unit tests? I would understand e. g. .csv as in- or output.

Reply:
These are used to test the file merging methods included in this pull request, see script_interface.py